gulp/ComponentSamples.js

```diff
@@ -69,21 +69,6 @@ gulp.task('ComponentSamples-moveJS', function() {
             .pipe(gulp.dest(Config.paths.distSamples + '/Components'));
 });
 
-// Style Linting
-// ----------------------------------------------------------------------------
-
-gulp.task('ComponentSamples-styleHinting',  function() {
```

Comment:
Interesting--is there no linting for SASS built in that we can switch to? We definitely want that included, very soon if not right now in this PR. I realize that's scope creep but it's an important aspect of development we shouldn't lose.